bin/asl-setup-dkms-mok

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/bash
+#
+
+if [ ${UID} -ne 0 ]; then
```

Comment:
${EUID} ?

Reply:
Since this isn't running under an suid or trying to drop privileges, UID and EUID will always be the same thing.